src/customize-pop/application-settings.md

```diff
@@ -0,0 +1,63 @@
+# Application Settings
+
+- [`Applications` settings](/customize-pop/application-settings.md#applications): Manage application permissions, allow system integration features, specify default handlers, and view disk space usage per app. These settings may not be available for all applications, depending on the functionality or access required by the application.
+- [`Notifications` settings](/customize-pop/application-settings.md#default-applications): Control which applications display notifications, what notifications are displayed, and where they are displayed.
+- [`Default Applications` settomgs](/customize-pop/application-settings.md#default-applications) Specify applications that should always open certain file types.
+
+---
+
+## Applications
+
+Options to control application access to system features and resources are located in the `Settings` ➞ `Applications` tab.
+
+![Application Settings](/images/application-settings/application-settings.png)
+
+| Option | Function |
+|---------|----------|
+| Permissions & Access | View the data and system services requested or required by an application. Permissions can include access to system devices, using the system's network connection, reading and writing to the file system, and the ability to change settings. Many applications include built-in permissions that cannot be modified by the user. View permission granted to an application by clicking the `Built-in Permissions` button. |
+| Integration | Disable or enable system-wide features used by an application. These features include the ability to appear in search, display notification, and running in the background. |
+| Default Handlers | This determines which file types are associated with an application. Click an entry to display more information about the file types assigned to the application. Multiple programs can be associated with the same file types. See the [Default Applications](/customize-pop/application-settings.md#default-applications) to specify an application that should always open a certain media type. |
+| Usage | Display an application's disk usage. Click on the `Storage` button to view space used by the application itself, data created by the application, and temporary data cached by the application. Click `Clear Cache` to clear an application's locally stored data.
+
+## Notifications
+
+Settings to control application notification behavior are located in the `Settings` ➞ `Notifications` tab.
+
+![Notifications Settings](/images/application-settings/notifications-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Do Not Disturb | Disable all application notifications globally. |
```

Comment:
This doesn't actually disable notifications, it just prevents them from showing as pop-ups. The notifications will still queue up in the notifications drop-down while this setting is enabled. It might be worth explaining that (so people know they will still be able to see the notifications later when using this option.)